astropy/cosmology/parameter/_descriptors.py

```diff
@@ -19,20 +19,52 @@
 
 @dataclass(frozen=True, **_dataclass_kwargs)
 class ParametersAttribute:
-    """Immutable mapping of the Parameters.
+    """Immutable mapping of the :class:`~astropy.cosmology.Parameter` objects or values.
 
-    If accessed from the class, this returns a mapping of the Parameter
-    objects themselves.  If accessed from an instance, this returns a
-    mapping of the values of the Parameters.
+    If accessed from the :class:`~astropy.cosmology.Cosmology` class, this returns a
+    mapping of the :class:`~astropy.cosmology.Parameter` objects themselves.  If
+    accessed from an instance, this returns a mapping of the values of the Parameters.
+
+    This class is used to implement the ``parameters`` attribute of
+    :class:`~astropy.cosmology.Cosmology`.
 
     Parameters
     ----------
     attr_name : str
         The name of the class attribute containing the Parameter objects.
```

Comment:
I guess this really should be "The name of the class attribute containing the *names* of the Parameter *attributes|descriptors*."?